verticapy/machine_learning/memmodel/tree.py

```diff
@@ -541,6 +605,70 @@ class BinaryTreeAnomaly(Tree):
         each class.
     psy: int, optional
         Sampling  size used to  compute the final  score.
+
+    Examples
+    --------
+
+    **Initalization**
+
+    Import the required module.
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.memmodel.tree import BinaryTreeAnomaly
+
+    A BinaryTreeAnomaly model is defined by its left and right child node id's,
+    feature and threshold value to split a node. Final values at leaf nodes are also required.
+    Let's create a :py:mod:`verticapy.machine_learning.memmodel.tree.BinaryTreeAnomaly` model.:
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.memmodel.tree import BinaryTreeAnomaly
+        children_left = [1, 3, None, None, None]
+        children_right = [2, 4, None, None, None]
+        feature = [0, 1, None, None, None]
+        threshold = ["female", 30, None, None, None]
+        value = [None, None, 3, 11, 1993]
+        model_bta = BinaryTreeAnomaly(children_left = children_left,
+                        children_right = children_right,
+                        feature = feature,
+                        threshold = threshold,
+                        value = value)
+
+    Create a dataset.
+
+    .. ipython:: python
+
+        data = [["male", 100], ["female", 20], ["female", 50]]
+
+    **Making In-Memory Predictions**
+
+    Use :py:meth:`verticapy.machine_learning.memmodel.tree.BinaryTreeAnomaly.predict`
+    method to do predictions
+
+    .. ipython:: python
+
+        #model_bta.predict(data)
```

Comment:
Commented as this was giving error. Need help on putting up a working example for  memmodel.tree.BinaryTreeAnomaly